Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Create_or_Edit_EDL_object.yml

```diff
@@ -406,16 +414,129 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: d3236cb3-33d2-4072-87d0-a78bfa8526ca
+    type: condition
+    task:
+      id: d3236cb3-33d2-4072-87d0-a78bfa8526ca
+      version: -1
+      name: Check if the EDL type is empty
+      description: Check if the EDL type is empty
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "1"
+      "yes":
+      - "17"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEmpty
+          left:
+            value:
+              simple: inputs.EDLObjectType
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 240,
+          "y": -560
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 8a1e23d8-217a-4560-8501-5369fcba3862
+    type: collection
+    task:
+      id: 8a1e23d8-217a-4560-8501-5369fcba3862
+      version: -1
+      name: Provide an EDL type
+      description: Provide an EDL type
+      type: collection
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -40,
+          "y": -390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: 'Choose one of the following types:'
+        required: true
+        gridcolumns: []
+        defaultrows: []
+        type: singleSelect
+        options: []
+        optionsarg:
+        - {}
+        - simple: domain
+        - simple: ip
+        - simple: url
+        - simple: predefined_ip
+        - simple: predefined_url
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Choose your EDL type
+      description: Your EDL type is missing and we want to ensure that your EDL object will be created successfully.
```

Comment:
Can you please can make this flow straight? 
![image](https://github.com/demisto/content/assets/61933087/eeb0b31e-28d7-4d17-b251-7c6c004e5b84)

Reply:
Done 
![Prisma_SASE_-_Create_or_Edit_EDL_object](https://github.com/demisto/content/assets/88268646/52ad6efe-037b-4cad-adae-c025dbc54cc0)